frontend/src/components/Research/Sampling/SamplingAlgoMain.jsx

```diff
@@ -0,0 +1,45 @@
+import { Container, Row } from "react-bootstrap";
+import Header from "../../Header/Header";
+import FormSamplingAlgo from "./FormSamplingAlgo";
+import StepButtons from "../Common/StepButtons";
+import { DataSelectorProvider } from "../../DataSelector/DataSelectorContext";
+
+// Array list sampling algorithms.
+const listSamplingAlgo = {
+    echantillonnage_aleatoire_simple: [
+        {name: "Nombre d'éléments aléatoire", type: "number"},
+    ],
+    echantillonnage_reservoir: [
+        {name: "Nombre d'éléments aléatoire", type: "number"},
+    ],
+    echantillonnage_systematique: [
+        {name: "Nombre d'éléments aléatoire", type: "number"},
+        {name: "Interval", type: "number"},
+    ],
+};
+
+/**
+ * Component for selecting and configuring sampling algorithms for two entities.
+ *
+ * @returns {JSX.Element} The JSX element representing the main sampling algorithm configuration component.
+ */
+function SamplingAlgoMain() {
```

Comment:
J'aime pas trop le nom du composant
```suggestion
function SamplingAlgorithmConfigurationStep {
```